security/src/test/java/io/airlift/security/csr/TestCertificationRequest.java

```diff
@@ -52,7 +52,6 @@ public void test()
         assertThat(certificationRequest.getCertificationRequestInfo()).isEqualTo(certificationRequestInfo);
         assertThat(certificationRequest.getSignatureAlgorithmIdentifier()).isEqualTo(signatureAlgorithmIdentifier);
         assertThat(base16().encode(certificationRequest.getSignature())).isEqualTo(base16().encode(signature));
-        assertThat(certificationRequest).isEqualTo(certificationRequest);
```

Comment:
Similar comment as above. This was intended to test that `.equals()` for that class is implemented correctly. It seems to have been lost in the migration to AssertJ